src/components/layout/form/utils/getRequiredValidate.js

```diff
@@ -1,9 +1,19 @@
 import { createValidateRequiredField } from 'react-final-form-utils'
 
+const createValidateRequiredBooleanField = errorMessage => {
+  return value => {
+    if (value === true || value === false) return undefined
```

Comment:
Je trouve que le undefined prête à confusion, par contre la logique m'échappe. 
un boolean c'est true ou false, jamais undefined...

```suggestion
    if (value === true || value === false) return
```

ce que tu dois check si je ne m'abuse, c'est qu'au moins un boolean dans un fieldset soit true, et donc qu'ils ne soient pas tous false.

Reply:
Non ici on check que la valeur est bien un boolean.
Si c'est le cas on retourne rien, sinon on retourne le message d'erreur "ce champs est obligatoire".